modules/ossm-cert-manager-verify.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-security.adoc
+
+
+:_content-type: PROCEDURE
+[id="ossm-cert-manager-verify_{context}"]
+== Verify cert-manager installation
+
+.Procedure
+
+. Deploy the HTTP and `sleep` apps:
++
+[source, terminal]
+----
+oc new-project foo
```

Comment:
IBM Style: foo noun
Do not use. Developers sometimes use this term as technical jargon in code and as shorthand for fubar, an acronym of profanity in code. If you see this term in sample code, revise the example or do not publish it.
Rewrite any text to omit the term. See also fubar.

Reply:
Maybe:
```suggestion
oc new-project <project-name>
```

Reply:
Multiple instances...